superset/advanced_data_type/plugins/oid_tag.py

```diff
@@ -0,0 +1,37 @@
+from superset.advanced_data_type.plugins import translate_filter_func
+from superset.advanced_data_type.plugins.operator_sets import EQUAL_NULLABLE_OPERATOR_SET, PATTERN_MATCHING_OPERATOR_SET
+from superset.advanced_data_type.types import AdvancedDataType, AdvancedDataTypeRequest, AdvancedDataTypeResponse
+
+
+def oid_tag_func(req: AdvancedDataTypeRequest) -> AdvancedDataTypeResponse:
+    """
+    Convert a passed in AdvancedDataTypeRequest to an AdvancedDataTypeResponse
+    """
+    resp: AdvancedDataTypeResponse = {
+        "values": [],
+        "error_message": "",
+        "display_value": "",
+        "valid_filter_operators": EQUAL_NULLABLE_OPERATOR_SET + PATTERN_MATCHING_OPERATOR_SET,
+    }
+
+    if req["operator"] in ['IS NULL', 'IS NOT NULL']:
+        return resp
+    elif req["values"] == [""]:
+        resp["error_message"] = "OID Tag must not be empty"
+        return resp
+    elif req["operator"] in ['LIKE', 'ILIKE']:
```

Comment:
I don't think OID Tag searches should be case-sensitive. Just `ILIKE`, no `LIKE`.

Reply:
8f89fb86f3fe898273a369955e6378e400477fcf